### PR TITLE
Cleanup test_runner

### PR DESCRIPTION
**Pull request by mellenburg**
Makes the code a little more sensible and fixes some documentation

### CODE REVIEW

**Comment by malnick** [comment]
👍

**Comment by kartick** [comment]
LGTM. Good cleanup action.

File: test_util/test_runner.py [hunk 1]
```diff
@@ -139,9 +138,9 @@ def integration_test(
         '-e', 'AWS_REGION='+region,
         '--net=host', '--name='+test_container_name, 'py.test', 'py.test',
         '-vv', ci_flags, '/integration_test.py']
-    log.info('Running integration test...')
     try:
-        with proxy_registry_as_local(tunnel, agent_list, test_dir):
+        with remote_port_forwarding(tunnel, agent_list, join(test_dir, 'ssh_Key')):
```
Comment (cmaloney) [inline test_util/test_runner.py]: `ssh_Key`?

**Comment by cmaloney** [comment]
The ssh_key -> ssh_Key seems suspect, other than that looks good

File: test_util/test_runner.py (hunk 1 above)
Comment (mellenburg) [inline test_util/test_runner.py]: typo. This is written such that this can be arbitrary path though

Comment (cmaloney) [inline test_util/test_runner.py]: Can we get the lower case / typo fixed?

**Comment by mellenburg** [comment]
@cmaloney This is ready to go

**Pull request merged**